theme/lumo/vaadin-grid-pro-edit-select.html

```diff
@@ -15,7 +15,7 @@
 
 <dom-module id="lumo-grid-pro-edit-select-text-field" theme-for="vaadin-select-text-field">
   <template>
-    <style include="lumo-select-text-field lumo-grid-pro-edit-text-field">
+    <style include="lumo-select-text-field lumo-grid-pro-edit-text-field-no-deps">
```

Comment:
Looks like you didn't replace this one. You can update with `[skip ci]` as build is green

Reply:
Damn, looks like I somehow missed it even though I was thinking about updating that too. Will fix it. Thanks!

Reply:
Done (amended and with [skip ci])